### ISSUE TITLE
UOS "inrange (serial) (range)" function doesn't work with items.

### ISSUE THREAD

**Issue by ngyoung** [jsebold666/RazorEnhanced#20]
Tested the following with an item and a mobile.  It works with a mobile but not an item:
```
//uos
unsetalias 'tobetamed'
promptalias 'tobetamed'
while not @inrange 'tobetamed' 1
 headmsg "Not in range!"
 pause 500
endwhile
headmsg 'You are in range'
```

**Comment by yoda29369** [comment]
agreed definately a problem...... same issues

**Comment by credzba** [comment]
I believe fixed in 7.7.28 / 8.0.10
